src/components/parameters.js

```diff
@@ -27,20 +26,20 @@ import {
 import { updateConfigParameter } from '../utils/rest-api';
 import { useSnackMessage } from '@gridsuite/commons-ui';
 
-const useStyles = makeStyles((theme) => ({
-    title: {
+export const styles = {
```

Comment:
no need to export this const
```suggestion
const styles = {
```